staging/knative/charts/eventing-sources/charts/github/templates/github.yaml

```diff
@@ -0,0 +1,1037 @@
+# Copyright 2019 The Knative Authors
```

Comment:
is the path to this file chosen purposely? Other components are located at:
- `staging/knative/charts/eventing/templates/eventing.yaml`
- `staging/knative/charts/serving/templates/serving.yaml`

Shall eventing-sources live in `staging/knative/charts/eventing-sourcing/templates/eventing-sourcing.yaml` instead?

Reply:
This is on purpose. There are many different types of event sources. E.g., Apache Kafka as an event source, CouchDB as an event source. Right now, we only add the github source.